build.sbt

```diff
@@ -585,43 +588,55 @@ lazy val allModules: Seq[ProjectReference] = Seq(
   http,
   kafka,
   `marshallers-jodatime`,
+  `legacy-avro-decimal-compat-model`,
+  `legacy-avro-decimal-compat-protocol`,
+  `legacy-avro-decimal-compat-encoders`,
+  `health-check-unary`,
+  `example-health-client`,
+  `example-health-server-monix`,
+  `example-health-server-fs2`,
   `example-routeguide-protocol`,
   `example-routeguide-common`,
   `example-routeguide-runtime`,
   `example-routeguide-server`,
   `example-routeguide-client`,
+  seed
+)
+
+lazy val otherModules: Seq[ProjectReference] = Seq(
+  `benchmarks-vprev`,
+  `benchmarks-vnext`,
   `example-todolist-protocol`,
   `example-todolist-runtime`,
   `example-todolist-server`,
-  `example-todolist-client`,
-  seed,
-  `benchmarks-vprev`,
-  `benchmarks-vnext`,
-  `legacy-avro-decimal-compat-model`,
-  `legacy-avro-decimal-compat-protocol`,
-  `legacy-avro-decimal-compat-encoders`,
-  `health-check-unary`,
-  `health-client`,
-  `health-server-monix`,
-  `health-server-fs2`
+  `example-todolist-client`
 )
 
-lazy val allModulesDeps: Seq[ClasspathDependency] =
-  allModules.map(ClasspathDependency(_, None))
+lazy val coreModulesDeps: Seq[ClasspathDependency] =
+  coreModules.map(ClasspathDependency(_, None))
 
 lazy val root = project
   .in(file("."))
   .settings(name := "mu-scala")
   .settings(noPublishSettings)
-  .aggregate(allModules: _*)
-  .dependsOn(allModulesDeps: _*)
+  .aggregate(coreModules: _*)
```

Comment:
Oh, root doesn't aggregate all the example projects any more? I'm not sure that's a good idea. I can imagine myself doing a refactoring, running `compile` or `test` and thinking everything's ok, but actually my refactoring has broken half the example projects.

Reply:
I think the amount of modules is currently insane. My proposal would be to move all the examples to its own repository.

```scala
lazy val nonCrossedScalaVersionModules: Seq[ProjectReference] = Seq(
  `benchmarks-vprev`,
  `benchmarks-vnext`,
  `example-todolist-protocol`,
  `example-todolist-runtime`,
  `example-todolist-server`,
  `example-todolist-client`
)
```

Currently, the `root` module cannot aggregate the modules above because they cannot be cross-compiled with Scala 2.13.x. That's why I split the aggregation into two different modules and added to Travis CI the proper check to make sure nothing is broken before going to master.

Reply:
Proposal filed at https://github.com/higherkindness/mu-scala/issues/817

Reply:
>I think the amount of modules is currently insane.

Agreed. It makes the CI painfully slow, and navigating the code is really difficult with the library and the examples all mixed together in the same repo. It makes the examples less discoverable as well.